src/log.c

```diff
@@ -375,7 +352,28 @@ int LogLoadEntries(Log *log)
 
             return RR_OK;
         }
+        /* validate crc for entry by reading and rebuilding it */
+        long read_crc;
```

Comment:
Suggest adding a dedicated function for entry validation.

Reply:
ok, can look at that.